configure.ac

```diff
@@ -1579,6 +1579,26 @@ else
   AC_MSG_RESULT([no])
 fi
 
+# Check for --enable-experimental-jit:
```

Comment:
Should this look for the LLVM tools and allow overriding the used paths and such?

Reply:
Maybe, but I'm personally *way* more comfortable with Python than Autoconf. This is basically the minimum that I could hack together to add the option.

Plus, that logic would need to be duplicated for PCbuild-based builds. Having the logic in the Python build script keeps it all in one easy-to-maintain place.

Reply:
Probably it works to just look up `$LLVM_<TOOL>` in os.environ first, or else take a `--llvm-<tool>=/path/to/executable`. You can then punt on wiring it up and let someone more comfortable with autoconf handle checking for it in configure.ac and passing / defining it in the Makefile.

I think the request isn't really about moving the special-cased searching to autoconf, just allowing autoconf to override it, however that may work out.

Reply:
I'll leave that to a future PR by somebody with a good use-case.

I'm trying to keep this one to just the required bits to actually JIT some code, since it's already a big, complex PR. We're already deferring more important stuff (like JIT code quality improvements) anyways.